docs/kubernetes/user-guides/driver-install.md

```diff
@@ -1,22 +1,53 @@
 # Kubernetes Driver Installation Guide
 
+## GCP Permissions Required
+
+The service account running `setup-project.sh` in order to set up a service
+account for the driver must have at least the following GCP IAM permissions:
+
+```
+iam.serviceAccounts.list
+iam.serviceAccountKeys.create
+iam.roles.create
+iam.roles.get
+iam.roles.update
+```
+
+If a service account of the name provided to `setup-project.sh` does not already
+exist the additional permissions are required:
+
+```
+resourcemanager.projects.getIamPolicy
+resourcemanager.projects.setIamPolicy
+iam.serviceAccounts.create
+iam.serviceAccounts.delete
+```
+
+These permissions are not required if you already have a service account ready
```

Comment:
Do we have instructions on how to use an existing service account and what permissions those need?

Reply:
added, PTAL